### PR TITLE
Bump Rancher to v2.7.9 and RKE2 to v1.26.11+rke2r1

### PR DESCRIPTION
**Pull request by bk201**
**Problem:**

We need to bump Rancher to v2.7.9 and RKE2 to v1.26.11+rke2r1.

**Solution:**
Bump Rancher to v2.7.9 and RKE2 to v1.26.11+rke2r1.

**Related Issue:**
https://github.com/harvester/harvester/issues/4782

**Test plan:**

### CODE REVIEW

**Comment by bk201** [comment]
New nodes can't join and Rancher keep complaing:
```
2023/11/17 08:55:44 [INFO] [planner] rkecluster fleet-local/local: non-ready bootstrap machine(s) custom-3eecdaaeac24 and join url to be available on bootstrap node
```

**Comment by FrankYang0529** [comment]
After few days investigation, the root cause may be that CAPI can't connect to rancher service. From Machine CR [1], its `NodeHealth` is false with `WaitingForNodeRef` reason. From [CAPI controller source code](https://github.com/kubernetes-sigs/cluster-api/blob/00dbf7b9f6322d7ebd06ae2efa703b23354dd37d/internal/controllers/machine/machine_controller_noderef.go#L52-L131), the Machine CR can pass `ProviderID` check, but it gets error in `r.Tracker.GetClient`, so the controller can't run remaining code to fix Machine CR (marking `NodeHealth` as true).

The error message in CAPI controller is:
```
"Reconciler error" err="failed to create cluster accessor: error creating client for self-hosted cluster: error creating dynamic rest mapper for remote cluster \"fleet-local/local\":
the server has asked for the client to provide credentials"
controller="machine" controllerGroup="cluster.x-k8s.io" controllerKind="Machine" Machine="fleet-local/custom-11a1422a6c10" namespace="fleet-local" name="custom-11a1422a6c10" reconcileID=77a0f08a-c816-4431-a2d9-e082c2d37064
```

Someone say the error is caused from different version between client-go and server [2]. In Rancher v2.7.9, it uses CAPI v1.4.4, CAPI uses controller-runtime v0.14.5, controller-runtime uses client-go v0.26.1. I tried to use different RKE2 versions like v1.26.1+rke2r1 or v1.26.10+rke2r2 with this Rancher update, but both can't fix the issue.

[1] Machine CR

```
apiVersion: cluster.x-k8s.io/v1beta1
kind: Machine
metadata:
  annotations:
    objectset.rio.cattle.io/applied: H4sIAAAAAAAA/5yTT3PTMBDFvwqzZzu15L/xDBcYTh04BIb7WlonIrLkkdYpTCffnbEpIWlpoD3aem/1232rexiIUSMjtPeAznlGNt7F+dN330hxJF4F41cKmS2tjL8xGlqY3IAOt6TTAdXOOILkWYO/cxTS7WEPLdwcRPLm1jj99jOpQPxPm8OBoAU1RfZDKgSKQkqslMj+yxpHVLPfeoUWjgmoQEuLX8xAkXEYoXWTtQlY7MgujSs7Raaw+p7umziXe/jxm+VXreQvOu84eJuOFt0s5DBdm8sO4w5ayIu6qmWmSYs+y8tcNiXKpirWFeZZjVRlWvX9WkACYU9n/mewHonOmdLg7RnYpZJY6auCh6TTZQHWpS67qsmwrJu+kE0lhawKWWdCrEXZ1LXO1lkptGwaRNkpKYuyyLtc9b3U+mnxOx/2FC7uPyZwNf3zfHtLxOkp5TiSmqPsvOfIAcclV+96s91Qvyz7aL5SiMY7aB+hHOZR742b29zcfnh3qvFanuNpVz5dRmVcHzBymBRPgV5G9n5h+Hh6fa9Gi4w8xYtpbQj1D2h7tJGeUv45Ox5/BgAA///dMbyVQgQAAA
    objectset.rio.cattle.io/id: unmanaged-machine
    objectset.rio.cattle.io/owner-gvk: /v1, Kind=Secret
    objectset.rio.cattle.io/owner-name: custom-11a1422a6c10
    objectset.rio.cattle.io/owner-namespace: local
    pre-terminate.delete.hook.machine.cluster.x-k8s.io/rke-bootstrap-cleanup: rke-bootstrap-controller
  creationTimestamp: "2023-12-12T15:27:40Z"
  finalizers:
  - machine.cluster.x-k8s.io
  generation: 3
  labels:
    cluster.x-k8s.io/cluster-name: local
    cluster.x-k8s.io/control-plane: "true"
    objectset.rio.cattle.io/hash: 3476720ded1f0353285a286496a307ae60dcff91
    rke.cattle.io/cluster-name: local
    rke.cattle.io/control-plane-role: "true"
    rke.cattle.io/etcd-role: "true"
    rke.cattle.io/machine-id: 95d5b680a578f42862126427011915877d09051d288aa2bc224543b3cff2dde
    rke.cattle.io/worker-role: "true"
  name: custom-11a1422a6c10
  namespace: fleet-local
  ownerReferences:
  - apiVersion: cluster.x-k8s.io/v1beta1
    kind: Cluster
    name: local
    uid: 0ae3d452-3dda-4aa2-855b-98dd57dd81e7
  resourceVersion: "3531"
  uid: a80a16c1-6313-45b5-a960-22118529f0da
spec:
  bootstrap:
    configRef:
      apiVersion: rke.cattle.io/v1
      kind: RKEBootstrap
      name: custom-11a1422a6c10
      namespace: fleet-local
    dataSecretName: custom-11a1422a6c10-machine-bootstrap
  clusterName: local
  infrastructureRef:
    apiVersion: rke.cattle.io/v1
    kind: CustomMachine
    name: custom-11a1422a6c10
    namespace: fleet-local
  nodeDeletionTimeout: 10s
  providerID: rke2://harvester-node-0
status:
  addresses:
  - address: 192.168.3.30
    type: InternalIP
  - address: harvester-node-0
    type: Hostname
  bootstrapReady: true
  conditions:
  - lastTransitionTime: "2023-12-12T15:27:40Z"
    status: "True"
    type: Ready
  - lastTransitionTime: "2023-12-12T15:27:40Z"
    status: "True"
    type: BootstrapReady
  - lastTransitionTime: "2023-12-12T15:27:40Z"
    status: "True"
    type: InfrastructureReady
  - lastTransitionTime: "2023-12-12T15:27:40Z"
    reason: WaitingForNodeRef
    severity: Info
    status: "False"
    type: NodeHealthy
  - lastTransitionTime: "2023-12-12T15:27:40Z"
    status: "True"
    type: PlanApplied
  - lastTransitionTime: "2023-12-12T15:27:54Z"
    status: "True"
    type: Reconciled
  infrastructureReady: true
  lastUpdated: "2023-12-12T15:27:54Z"
  observedGeneration: 2
  phase: Provisioned
```

[2] https://stackoverflow.com/questions/76177011/error-validating-data-the-server-has-asked-for-the-client-to-provide-credential

File: scripts/images/rancher-images.txt [hunk 1]
```diff
@@ -16,14 +16,13 @@ docker.io/rancher/mirrored-prometheus-operator-prometheus-config-reloader:v0.59.
 docker.io/rancher/mirrored-prometheus-operator-prometheus-operator:v0.59.1
 docker.io/rancher/mirrored-prometheus-prometheus:v2.38.0
 docker.io/rancher/mirrored-prometheus-alertmanager:v0.24.0
-docker.io/rancher/rancher-webhook:v0.3.5
-docker.io/rancher/rancher:v2.7.5
-docker.io/rancher/shell:v0.1.8
-docker.io/rancher/shell:v0.1.18
```
Comment (FrankYang0529) [inline scripts/images/rancher-images.txt:22]: The job `cattle-monitoring-system/rancher-monitoring-crd-create` needs `docker.io/rancher/shell:v0.1.18`.

**Review by FrankYang0529 — CHANGES_REQUESTED** [review]

File: scripts/images/rancher-images.txt (hunk 1 above)
Comment (bk201) [reply scripts/images/rancher-images.txt:22]: nice catch! Thanks!

**Review by FrankYang0529 — APPROVED** [review]
LGTM. I can upgrade from v1.2.1 and provision in airgap environment.

**Review by ibrokethecloud — APPROVED** [review]
lgtm. thanks.

**Comment by FrankYang0529** [comment]
@mergify backport v1.2

**Pull request merged**